src/main/java/discord4j/discordjson/json/RoleData.java

```diff
@@ -18,17 +20,17 @@
 
     String name();
 
-    int color();
+    Integer color();
```

Comment:
Why did you change this? If there is no reason, primitives (boolean, int, long...) are preferable over objects

Reply:
My IDE didn't like it for some weird reason. I forgot to change it back when making this PR sorry.